sql/base/dbc/cmangos_fixes/Spell.sql

```diff
@@ -207,6 +207,12 @@ UPDATE `spell_template` SET `RangeIndex` = 0 WHERE `Id` IN (16613,16619,16630,16
 -- Toxic Gas - used by Garden Gas in Naxx, remove SPELL_ATTR_EX_CHANNELED_1 and CHANNEL_FLAG_MOVEMENT to prevent aura being removed
 UPDATE spell_template SET AttributesEx=0, ChannelInterruptFlags=0 WHERE Id=30074;
 
+-- Lunar Festival fireworks spells have no missile speed serverside
+UPDATE `spell_template` SET `speed`=0 WHERE `Id` IN (26286, 26291, 26292, 26294, 26295, 26333, 26334, 26336, 26337, 26338, 26304, 26325, 26327, 26328, 26329, 26488, 26490, 26517, 26519, 26518, 26521);
```

Comment:
Now I realized something. Do you have a sniff of these being casted by any chance? The only reason why I am asking is because speed is ignored for GO casted spells. Which would explain the bad data and lack of "use" in the original spell because a GO cannot use another GO.

Reply:
Nvm, I rechecked, and my asumption is bogus :D

Reply:
[Here's the relevant packets for one of those spells](https://pastebin.com/K3CEz1S8), just in case :P

The effect is executed instantly, and it is even visible (if you use the firework as far as possible from the launcher, the launcher will trigger before the firework projectile reaches it. Strange, but blizzlike :D).